fuzzers/005-tilegrid/clk_bufg/generate.tcl

```diff
@@ -0,0 +1,21 @@
+source "$::env(XRAY_DIR)/utils/utils.tcl"
```

Comment:
Maybe it is worth moving this tcl script to a `common` directory as many different fuzzers in 005-tilegrid share it?

Reply:
Done